micro_ros_setup/config/zephyr/generic/create.sh

```diff
@@ -0,0 +1,52 @@
+# To be removed later, please use package.xml
+sudo apt update
+sudo apt install --no-install-recommends git cmake ninja-build gperf ccache dfu-util device-tree-compiler wget python3-pip python3-setuptools python3-tk python3-wheel xz-utils file make gcc gcc-multilib software-properties-common -y
+
+wget -O - https://apt.kitware.com/keys/kitware-archive-latest.asc 2>/dev/null | sudo apt-key add -
+sudo apt-add-repository 'deb https://apt.kitware.com/ubuntu/ bionic main'
+sudo apt update
+sudo apt install cmake -y
+
+# pip3 install --user -U west
+export PATH=~/.local/bin:"$PATH"
+
+pushd $FW_TARGETDIR >/dev/null
+   
+    west init zephyrproject
+    pushd zephyrproject >/dev/null
+        west update
+    popd >/dev/null
+
+    pip3 install -r zephyrproject/zephyr/scripts/requirements.txt
+
+    export TOOLCHAIN_VERSION=zephyr-toolchain-arm-0.11.2-setup.run
```

Comment:
Can we configure the toolchain to use somewhere?

Reply:
Where do you propose to configure it?

Reply:
No idea. Let's leave it at this, then.